css/backend/src/main/scala/io/udash/css/CssFileRenderer.scala

```diff
@@ -0,0 +1,40 @@
+package io.udash.css
+
+import java.io.{File, PrintWriter}
+
+import scalacss.internal.Renderer
+
+/**
+  * Renders provided styles into files. Creates separate file for each stylesheet.
+  * Keeps styles order from provided `Seq`.
+  * @param dirPath Target directory for rendered files.
+  * @param styles Sequence of stylesheets.
+  * @param createMain If true, creates `main.css` file. It imports all other stylesheets.
+  */
+class CssFileRenderer(dirPath: String, styles: Seq[_ <: CssBase], createMain: Boolean) {
```

Comment:
You don't need to use an upper-bound for `styles`, `Seq[CssBase]` should be enough.